docs/src/data/nav-items.yaml

```diff
@@ -20,8 +20,10 @@
       path: /guides/FHIRValidationGuide
     - title: Terminology guide
       path: /guides/FHIRTerminologyGuide
-    - title: Create the fhir-validation assembly
+    - title: Create the Validation assembly
       path: /guides/CreateFHIRValidationAssembly
+    - title: Performance guide
+      path: /guides/FHIRPerformanceGuide/
```

Comment:
I noticed we're a little inconsistent on whether we include the trailing / or not